Doc/conf.py

```diff
@@ -23,6 +23,10 @@
 except ImportError:
     _tkinter = None
 '''
+
+# Manpage config
+manpages_url = os.getenv('MANPAGES_URL', 'https://manpages.debian.org/{path}')
```

Comment:
I don't think this is something that needs to be customized via an environment variable.

Setting its value directly should be enough IMO:

```py
manpages_url = 'https://manpages.debian.org/{path}'
```

Reply:
Done